### DISCUSSION TITLE
配置443访问apollo,为啥登录跳转之后，变成了到http了导致跳转失败

### DISCUSSION THREAD

**Discussion by feixia5712** [apolloconfig/apollo#3794] [Show and tell]
nginx配置如下
server {
     listen       8070 ssl;
     server_name  localhost;
     ssl_certificate     cert/apollo/apollo-cert.pem;
     ssl_certificate_key cert/apollo/apollo-key.key;

     ssl_session_cache    shared:SSL:1m;
     ssl_session_timeout  5m;

     ssl_ciphers "HIGH+EECDH:HIGH+EDH:!aNULL:!MD5:!SHA";
     ssl_prefer_server_ciphers  on;
     ssl_protocols TLSv1.2;
     if ($time_iso8601 ~ "^(\d{4})-(\d{2})-(\d{2})") {
     set $year $1;
     set $month $2;
     set $day $3;
     }
    access_log  logs/https-apollo-$year-$month-$day-access.log main;
    error_log  logs/https-apollo-error.log  notice;

location / {
proxy_pass http://192.168.1.180:8070;
proxy_set_header Host $host;
proxy_set_header X-Real-IP $remote_addr;
proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
proxy_set_header X-Forwarded-Proto "https";
}

}

浏览器访问
https://ip:8070/signin可以是登录页面
但是输入密码之后，跳转失败
![image](https://user-images.githubusercontent.com/12670758/123773746-753b1b00-d8ff-11eb-9938-84a01652efea.png)
请问下该如何配置

**Comment by vdiskg** [comment]
host 的配置需要改成这样
```nginx
proxy_set_header Host $http_host;

```
